.cirrus.yml

```diff
@@ -157,7 +157,7 @@ build_and_release_dotnet_task:
   restore_and_build_script:
     # Sets the $BUILD_NUMBER; used for setting the project version.
     # See https://github.com/SonarSource/re-ci-images/blob/master/docker/bin/cirrus-env#L28
-    - source cirrus-env QA
```

Comment:
This change will break the build. Why do you need it?

Reply:
Discussed offline, mentioning also here for the record: I commented out the rest of the tasks and the single task does not work without this change.
It will be reverted before asking for review.